src/directives/geojson.js

```diff
@@ -32,7 +32,7 @@ angular.module("leaflet-directive")
                         onEachFeature = geojson.onEachFeature;
                     } else {
                         onEachFeature = function(feature, layer) {
-                            if (leafletHelpers.LabelPlugin.isLoaded() && isDefined(geojson.label)) {
+                            if (leafletHelpers.LabelPlugin.isLoaded() && isDefined(feature.properties.description)) {
```

Comment:
According to the [geojson spec](http://geojson.org/geojson-spec.html#feature-objects), a feature should always have properties, but I guess not all geojsons in the wild adhere to that spec.

So yeah, adding `isDefined(feature.properties)` is a good idea.